packages/image-minify/index.js

```diff
@@ -1,49 +1,26 @@
 const merge = require('deepmerge');
-const ImageminWebpackPlugin = require('imagemin-webpack/ImageminWebpackPlugin');
-const gifsicle = require('imagemin-gifsicle');
-const svgo = require('imagemin-svgo');
-const pngquant = require('imagemin-pngquant');
-const mozjpeg = require('imagemin-mozjpeg');
+const ImageminWebpackPlugin = require('imagemin-webpack-plugin').default;
 const webp = require('imagemin-webp');
 
-const imageminLoader = require.resolve('imagemin-webpack/imagemin-loader');
-
 module.exports = (neutrino, opts = {}) => {
   const options = merge({
     imagemin: {
-      plugins: [
-        gifsicle(),
-        svgo(),
-        pngquant(),
-        mozjpeg(),
-        webp()
-      ]
-    },
-    plugin: {
-      name: '[path][name].[ext]'
+      plugins: [],
+      optipng: {},
+      gifsicle: {},
+      jpegtran: {},
+      svgo: {},
+      pngquant: null,
+      webp: {}
     },
```

Comment:
Is there a reason for setting these all to the same defaults they already have, or is this just for clarity? https://github.com/Klathmon/imagemin-webpack-plugin#optionsoptipng